server/api/reviews.go

```diff
@@ -0,0 +1,133 @@
+package handler
+
+import (
+	"net/http"
+	"time"
+
+	"github.com/codeday-labs/bookworms/server/db"
+	"github.com/codeday-labs/bookworms/server/utils"
+	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/bson/primitive"
+)
+
+type ReviewBody struct {
+	UserName   string   `json:"user_name"`
+	BookName   string   `json:"book_name"`
+	Text       string   `json:"text"`
+	Categories []string `json:"categories"`
+}
+
+// singular collection name from mongodb collections conventions
+const ReviewsCollection = "review"
+
+func filterReviews(filter interface{}) ([]*db.Review, error) {
+
+	// a slice to store decoded reviews
+	var reviews []*db.Review
+
+	DB, err := db.DB()
+
+	if err != nil {
+		return nil, err
+	}
+
+	cur, err := DB.Collection(ReviewsCollection).Find(db.Ctx, filter)
+
+	// close db connection
+	defer DB.Client().Disconnect(db.Ctx)
```

Comment:
This line should move to line 29 to close the connection even when there is an error.